x-pack/plugins/uptime/public/components/monitor/synthetics/step_detail/waterfall/types.ts

```diff
@@ -111,6 +111,7 @@ export const FriendlyMimetypeLabels = {
 export const MimeTypesMap: Record<string, MimeType> = {
   'text/html': MimeType.Html,
   'application/javascript': MimeType.Script,
+  'application/json': MimeType.Script,
```

Comment:
We could add json to javascript, have it just be an "other" mime type, or create a new json type with its own legend item.

Reply:
Personally I'm fine with this. cc @shahzad31